config/environments/development.rb

```diff
@@ -71,7 +76,8 @@
 
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
-  # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+  config.file_watcher = ActiveSupport::EventedFileUpdateChecker
```

Comment:
Note: this doesn't work in docker. For docker we need to use `ActiveSupport::FileUpdateChecker`. We often use an `ENV` check to see if we're in docker mode.